sci-physics/fastjet/fastjet-3.4.0.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{8..10} )
+DOCS_BUILDER="doxygen"
+
+inherit autotools flag-o-matic fortran-2 python-r1 docs
+
+DESCRIPTION="A software package for jet finding in pp and e+e- collisions"
+HOMEPAGE="https://fastjet.fr/"
+SRC_URI="https://fastjet.fr/repo/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="cgal doc examples python +plugins +contrib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# For now, we depend on cgal[gmp(-)] and append -lgmp to libs, despite possible incompatibility with cgal-5.5
+DEPEND="
+	contrib? ( sci-physics/fastjet-contrib )
+	cgal? ( sci-mathematics/cgal:=[shared(+),gmp(-)] )
+	plugins? ( sci-physics/siscone:= )
+	python? ( ${PYTHON_DEPS} )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	doc? ( app-doc/doxygen[dot] )
+	virtual/fortran
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-system-siscone.patch
+	"${FILESDIR}"/${P}-gfortran.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	append-libs -lgmp
```

Comment:
This should still be behind `use cgal` I think. It is only the `has_version` check that seems weird to me.